bot/exts/utilities/bookmark.py

```diff
@@ -134,24 +186,41 @@ def event_check(reaction: discord.Reaction, user: discord.Member) -> bool:
                     user.id != self.bot.user.id
                 ))
             )
-        await self.action_bookmark(ctx.channel, ctx.author, target_message, title)
-
-        # Keep track of who has already bookmarked, so users can't spam reactions and cause loads of DMs
-        bookmarked_users = [ctx.author.id]
-        reaction_message = await self.send_reaction_embed(ctx.channel, target_message)
 
         while True:
             try:
-                _, user = await self.bot.wait_for("reaction_add", timeout=TIMEOUT, check=event_check)
+                reaction, user = await self.bot.wait_for("reaction_add", timeout=TIMEOUT, check=event_check)
             except asyncio.TimeoutError:
                 log.debug("Timed out waiting for a reaction")
                 break
             log.trace(f"{user} has successfully bookmarked from a reaction, attempting to DM them.")
             await self.action_bookmark(ctx.channel, user, target_message, title)
+            await reaction.remove()
             bookmarked_users.append(user.id)
 
         await reaction_message.delete()
 
+    @commands.dm_only()
+    @bookmark.command(name="delete", aliases=("del",), root_aliases=("unbm", "unbookmark"))
```

Comment:
The extra `,` was present so that `("del")` remains a tuple, otherwise, python would consider it as a single string. Trailing commas are only used for multiline tuples or single length tuples IIRC.

Reply:
forcepushed onto https://github.com/python-discord/sir-lancebot/pull/929/commits/6fcefb0a6b2153731b72d7ce194531170c5fcd3e